packages/autorest.python/autorest/codegen/serializers/builder_serializer.py

```diff
@@ -755,12 +755,13 @@ def _serialize_body_parameter(self, builder: OperationType) -> List[str]:
         body_param = cast(BodyParameter, builder.parameters.body_parameter)
         if body_param.is_form_data:
             return [
+                'file_properties = ["' + '", "'.join(body_param.file_properties) + '"]',
```

Comment:
all we should do for file_properties is really just combine all of the `data` and `files` entries that users have inputted into a list of tuples, ten pass that to the `files` kwarg of `HttpRequest`

Reply:
Updated